src/main/java/com/ts/tserp/model/User.java

```diff
@@ -1,10 +1,70 @@
 package com.ts.tserp.model;
 
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 
-@Entity
+@Entity(name="user")
 public class User {
 
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
 	private int id;
+	private String first_name;
+	private String last_name;
+	private int phone_no;
```

Comment:
Add unique constraint to phone_number. Add a regular expression to evaluate phone number